INSTALL.rst

```diff
@@ -0,0 +1,68 @@
+..
+    Copyright (C) 2018 CERN.
+
+    Asclepias Broker is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Installation
+============
+
+To run an instance of ``Asclepias Broker`` you will need a PostgreSQL and Elasticsearch, which can be installed directly on your machine,
+or you can use ``docker-compose`` with the provided docker configuration to run those in the sandboxed docker environment (recommended).
+
+Quick start
+-----------
+
+.. note::
+
+    The docker configuration provided in this module uses PostgresSQL and ElasticSearch 6.
+
+Install ``docker`` and ``docker-compose`` in your machine.
+Then, create and run the services using the docker containers:
```

Comment:
It would be good to mention briefly what services are being run in the docker container. I think this may be what the note above was doing, but it doesn't say if any other services are being run.